whois-scheduler/src/main/java/net/ripe/db/whois/scheduler/task/grs/ArinGrsSource.java

```diff
@@ -74,8 +81,15 @@ public void acquireDump(final Path path) throws IOException {
     }
 
     @Override
-    public void handleObjects(final File file, final ObjectHandler handler) throws IOException {
+    public void acquireIrrDump(final Path path) throws IOException {
+        if (Strings.isNullOrEmpty(irrDownload)) {
+            return;
+        }
+        downloader.downloadTo(logger, new URL(irrDownload), path);
```

Comment:
new URL(String) is deprecated
Could you use new URI(String).toURL() instead?

Reply:
This is *existing* code that i didn't want to refactor